groups/sig-cluster-lifecycle/groups.yaml

```diff
@@ -480,6 +480,22 @@ groups:
       - dlipovetsky@d2iq.com
       - neolit123@gmail.com
 
+  - email-id: sig-cluster-lifecycle-kubespray-alerts@kubernetes.io
+    name: sig-cluster-lifecycle-kubespray-alerts
+    description: |-
+      Group for Kubespray maintainers to receive CI system notifications.
+    settings:
+      WhoCanDiscoverGroup: "ANYONE_CAN_DISCOVER"
+      WhoCanViewGroup: "ANYONE_CAN_VIEW"
+      WhoCanPostMessage: "ANYONE_CAN_POST"
+      WhoCanJoin: "ANYONE_CAN_JOIN"
+      MessageModerationLevel: "MODERATE_ALL_MESSAGES"
+      ReconcileMembers: "false"
+    owners:
+      - kay.yan@daocloud.io
+      - mg@max.gautier.name
+      - tico88612@gmail.com
```

Comment:
I think you should add @ant31 as well (2t.antoine@gmail.com)

Reply:
Thanks @VannTen 
The email has been added